src/nvim/version.c

```diff
@@ -57,7 +57,6 @@ static char *(features[]) = {
   "+cursorbind",
   "+cursorshape",
   "+dialog_con",
-  "+diff",
```

Comment:
@Hinidu I just wanted to keep this PR well contained, since I already removed all references in the docs to `+diff`. I do agree with you though, and I already have a feature branch for that here:
https://github.com/Pyrohh/neovim/commits/cleanup-version-output

The output might need some work though, as it looks a bit odd/incomplete:
```
NVIM 0.0.0-alpha+201501201544 (compiled Jan 20 2015 15:44:59)
Commit: 144c3fc0587ead7f51b9c02438f2a10e5747776c
Build type: Debug
Compilation: /usr/bin/cc -Wconversion -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -DINCLUDE_GENERATED_DECLARATIONS -DHAVE_CONFIG_H -I/home/michael/src/neovim/build/config -I/home/michael/src/neovim/src -I/home/michael/src/neovim/.deps/usr/include -I/home/michael/src/neovim/.deps/usr/include -I/home/michael/src/neovim/.deps/usr/include/luajit-2.0 -I/usr/include -I/home/michael/src/neovim/build/src/nvim/auto -I/home/michael/src/neovim/build/include
Compiled by michael@orchid  Features included (+) or not (-):
+acl      +iconv    +terminfo 
   system vimrc file: "$VIM/nvimrc"
     user vimrc file: "$HOME/.nvimrc"
 2nd user vimrc file: "~/.nvim/nvimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/local/share/nvim"
```